.github/workflows/build.yml

```diff
@@ -56,22 +55,42 @@ jobs:
             --manifest-path=script/rust/Cargo.toml
             --bin test-all
             -- --headless --firefox
-
-# TODO[AO] formatter does not work
-#  fmt:
-#    name: Formatter
-#    runs-on: ubuntu-latest
-#    steps:
-#      - uses: actions/checkout@v1
-#      - uses: actions-rs/toolchain@v1
-#        with:
-#          toolchain: nightly-2019-11-04
-#          override: true
-#      - run: rustup component add rustfmt
-#      - uses: actions-rs/cargo@v1
-#        with:
-#          command: fmt
-#          args: --all -- --check
+  # TODO[AO] formatter does not work
+  #  fmt:
+  #    name: Formatter
+  #    runs-on: ubuntu-latest
+  #    steps:
+  #      - uses: actions/checkout@v1
+  #      - uses: actions-rs/toolchain@v1
+  #        with:
+  #          toolchain: nightly-2019-11-04
+  #          override: true
+  #      - run: rustup component add rustfmt
+  #      - uses: actions-rs/cargo@v1
+  #        with:
+  #          command: fmt
+  #          args: --all -- --check
+  size:
+    name: Check size
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install Rust Nightly Toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly-2019-11-04
+          override: true
+      - name: Install wasm-pack
+        uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: wasm-pack --version 0.8.1
+      - name: Compile WASM
```

Comment:
Strictly speaking it is not "Compile WASM" , this is "Compile to WASM", or "Compile Rust to WASM". I know the difference is subtle, but lets make it correct.